src/main/resources/messages_es.properties

```diff
@@ -0,0 +1 @@
+email.unsubscribe.footer=Para darse de baja haga clic: <%unsubscribe_url%>
```

Comment:
Minor question: this url has <angle brackets> but the default file does not. Does that matter?